mock/syfooversiktsrv/mockSyfooversiktsrv.ts

```diff
@@ -0,0 +1,23 @@
+import { SYFOOVERSIKTSRV_PERSONTILDELING_ROOT } from "../../src/apiConstants";
+import { NAV_PERSONIDENT_HEADER } from "../util/requestUtil";
+import express from "express";
+
+export const mockSyfooversiktsrv = (server: any) => {
+  server.get(
+    `${SYFOOVERSIKTSRV_PERSONTILDELING_ROOT}/persontildeling/personer/single`,
+    (req: express.Request, res: express.Response) => {
+      if (req.headers[NAV_PERSONIDENT_HEADER]?.length === 11) {
+        res.setHeader("Content-Type", "application/json");
+        res.send(JSON.stringify(veilederBrukerKnytningMock));
+      } else {
+        res.status(400).send("Did not find PersonIdent in headers");
+      }
+    }
+  );
+};
+
+const veilederBrukerKnytningMock = {
+  personident: "12345678910",
```

Comment:
Kan være greit å bruke verdier for personident, veilederident og enhet fra `mockConstants.ts` her så henger det mer sammen med andre mockdata.